Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/SevenDayRValueCard.kt

```diff
@@ -32,12 +38,38 @@ class SevenDayRValueCard(parent: ViewGroup) :
         }
 
         with(item.stats as SevenDayRValue) {
+
+            sevenDayRValueContainer.contentDescription =
+                buildAccessibilityStringForSevenDayRValueCard(item.stats, reproductionNumber)
+
             primaryLabel.text = getPrimaryLabel(context)
             primaryValue.text = getLocalizedSpannableString(
                 context,
                 formatStatisticalValue(context, reproductionNumber.value, reproductionNumber.decimals)
             )
+
+            primaryValue.contentDescription = StringBuilder()
+                .appendWithWhiteSpace(context.getString(R.string.statistics_title_reproduction))
+                .appendWithWhiteSpace(getPrimaryLabel(context))
```

Comment:
very minor: maybe rename to `appendingWithTrailingSpace()`

Reply:
Done ✔️